api/CanvasRenderingContext2D.json

```diff
@@ -0,0 +1,3857 @@
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "This feature is deactivated by default. Enable the <code>ExperimentalCanvasFeatures</code> flag to test it."
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "fillStyle": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/fillStyle",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "filter": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CanvasRenderingContext2D/filter",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "52"
```

Comment:
I suspect that it should be 52 for both webview and chrome android. I'm verifying this with the [project owners](https://www.chromestatus.com/features/6701649301078016) and will get back to you.

Rule of thumb: if Webview and Chrome Android version numbers are missing from a Chrome Status entry, it's more likely that the project owner neglected to fill in the boxes than that it is that the feature is not supported. Email the owner if you have questions. That's why email addresses are on Chrome Status.

Reply:
Or contact me through my GitHub account.

Reply:
OK @jpmedley - thanks. Waiting to hear back from you on this one before making the appropriate changes and pushing.

Reply:
I just pinged them again. If I don't get an answer before @Elchi3 finishes his review you are free to merge and I'll submit a separate PR later.

Reply:
I just got an answer. This is in both Chrome Android and Android Webview.

Reply:
Updated.